po/da.po

```diff
@@ -1107,6 +1125,10 @@ msgid ""
 "mutexes) and async/await concurrency (cooperative multitasking using "
 "futures)."
 msgstr ""
+"[Samtidighed](concurrency.md): en hel kursusdag om samtidighed "
+"(_concurrency_) i Rust. Vi dækker både klassisk samtidighed (preventiv "
```

Comment:
I can see how preemptive can (in general) translate to preventiv but I'm not convinced it works in context of multitasking since the running thread gets yanked off the CPU in response to some timer event or interrupt. 

Alternatives

> ægte multitasking

I've seen that one used to describe preemptive multitasking but it is more nebulous/imprecise. 

> tidsdelt multitasking

I came across this one by reading what how this is translated into Norwegian 😺  I think it strikes a nice balance between being technically accurate and natural in Danish.